src/lib.rs

```diff
@@ -19,12 +19,12 @@
 //! system for cloud native applications, `IoT` messaging, and microservices
 //! architectures.
 //!
-//! For async API refer to the [`async-nats`] crate.
+//! For async API refer to the [`asynk`] module.
 //!
 //! For more information see [https://nats.io/].
 //!
 //! [https://nats.io/]: https://nats.io/
-//! [`async-nats`]: https://docs.rs/async-nats
+//! [`async`]: crate::async
```

Comment:
Rust doc does not render these lines as links, they used for defining where the links point to.

Reply:
My bad :).
The module though is named `asynk`.

Reply:
Ah not sure how I missed that, but turns out linking to a module does not require a link declaration.